### DISCUSSION TITLE
Two Renovent HR

### DISCUSSION THREAD

**Discussion by vvhillo** [tijsverkoyen/Home-Assistant-BrinkRenoventHR#5] [Q&A]
I have two Renovent HR (Medium) what do I need to change in the code so Home Assistant sees two Renovents? If I do it now it is a mixture of the two...

**Comment by tijsverkoyen** [comment]
Probably the `uniq_id` should be alterted when uploading to the second Wemos D1

For instance `entity["uniq_id"] = "brink_supply_in_temperature";` should be changed to `entity["uniq_id"] = "brink2_supply_in_temperature";`

And this will need to happen for all `uniq_id` in the ino file.

**Comment by vvhillo** [comment]
Thanks for the quick reply! Really like this solution you made!

**Comment by tijsverkoyen** [comment]
https://github.com/raf1000/brink_openhab deserves a lot of credit ;-)

**Comment by vvhillo** [comment]
It looks like I have to change a lot more than only the uniq_ids somehow.

**Comment by vvhillo** [comment]
I also had to change all of these: MqttPublishAutoDiscoveryMessage("homeassistant/sensor/brink_**2**__wifi_rssi/config", entity);
